mtags/src/main/scala-3/scala/meta/internal/pc/printer/MetalsPrinter.scala

```diff
@@ -255,7 +280,18 @@ class MetalsPrinter(names: ShortenedNames, dotcPrinter: DotcPrinter)(using
       // print only type string
       // e.g. "using Ord[T]" instead of "using x$0: Ord[T]"
       paramTypeString
-    else s"${keywordName}: ${paramTypeString}"
+    else
+      val isDefaultParam = param.isAllOf(DefaultParameter)
+      val default =
+        if includeDefaultParam && isDefaultParam then
+          val defaultValue = defaultValues.lift(index) match
+            case Some(value) if includeDefaultParam && !value.isEmpty => value
+            case _ => "{}"
+          s" = $defaultValue"
+        // to be populated later, otherwise we would spend too much time in completions
```

Comment:
We don't want to calculate defaults for anything that is not selected, it causes to bring in the docs.